packages/cli-kit/src/string.ts

```diff
@@ -39,15 +40,18 @@ export function capitalize(string: string) {
 }
 
 /**
- * Given a store, returns a valid store fqdn removing protocol and adding .myshopify.com domain
+ * Given a store, returns a valid store fqdn removing protocol and adding the proper domain in case is missing
  * @param store - Original store name provided by the user
  * @returns a valid store fqdn
  */
 export function normalizeStoreName(store: string) {
```

Comment:
It feels like `normalizeStoreName` is moving beyond a `string` operation. Is there a better place for this to live?

Reply:
agree